workflows/pipe-common/scripts/send_statistics.py

```diff
@@ -0,0 +1,394 @@
+    usage_costs = _get_usage_costs(runs)
+    logger.info('Usage costs: {}.'.format(usage_costs))
+    usage_weight = usage_costs / platform_usage_costs
+    logger.info('Usage weight: {}.'.format(usage_weight))
+    clusters_compute_hours = _get_cluster_compute_hours(api, from_date_time, to_date_time, user)
+    logger.info('Clusters compute hours: {}.'.format(clusters_compute_hours))
+    worker_nodes_count = _get_worker_nodes_count(api, from_date_time, to_date_time, user)
+    logger.info('Clusters worker nodes count: {}.'.format(worker_nodes_count))
+    login_time = _get_login_time(api, from_date_time, to_date_time, user)
+    logger.info('Login time: {}.'.format(login_time))
+    top3_instance_types = _get_top3(runs, "Instance")
+    logger.info('Top 3 Instance types: {}.'.format(top3_instance_types))
+    top3_pipelines = _get_top3(runs, "Pipeline")
+    logger.info('Top 3 Pipelines: {}.'.format(top3_pipelines))
+    top3_tools = _get_top3(runs, "Tool")
+    logger.info('Top 3 Tools: {}.'.format(top3_tools))
+    top3_used_buckets = _get_used_buckets(api, from_date_time, to_date_time, user, storages)
+    logger.info('Top 3 Used buckets: {}.'.format(top3_used_buckets))
+    top3_run_capabilities = _get_top3_run_capabilities(api, from_date_time, to_date_time, user, capabilities)
+    logger.info('Top 3 Capabilities: {}.'.format(top3_run_capabilities))
+    return Stat(compute_hours, runs_count, storage_read, storage_write, usage_weight, login_time, cpu_hours,
+                gpu_hours, clusters_compute_hours, worker_nodes_count,
+                top3_instance_types, top3_pipelines, top3_tools, top3_used_buckets, top3_run_capabilities)
+
+
+def _get_login_time(api, from_date_time, to_date_time, user):
+    users_report = api.report_users(from_date_time, to_date_time, [user])
+    active_hours_list = [1 for d in users_report if d.get('activeUsers') is not None]
+    return len(active_hours_list)
+
+
+def _get_platform_usage_cost(api, from_date, to_date):
+    all_run_dict = _get_runs(api, from_date, to_date, None)
+    return _get_usage_costs(all_run_dict)
+
+
+def _get_runs(api, from_date, to_date, user):
+    filters = {} if user is None else {"owner": [user]}
+    run_data = api.billing_export(from_date, to_date, filters, ["RUN"])
+    return _read_content(run_data, False)
+
+
+def _get_storage_usage(api, from_date_time, to_date_time, user, usage):
+    filter = {'messageTimestampFrom': from_date_time, 'messageTimestampTo': to_date_time, 'types': ['audit'],
+              'users': [user], 'message': usage}
+    storage_usage = api.log_group(filter, "user")
+    if storage_usage is None or storage_usage.get(user) is None:
+        return 0
+    return storage_usage.get(user)
+
+
+def _get_top3_run_capabilities(api, from_date, to_date, user, capabilities):
+    capabilities_dict = {}
+    for c in capabilities:
+        runs = api.filter_runs(from_date, to_date, user, {'partialParameters': CAPABILITY_TEMPLATE.format(c)})
```

Comment:
To get list of runs matching the filter let's not use MAX_PAGE_SIZE instead add a loop fetching runs page by page (e.g. 100 runs at once)